freedom/reco/postfit.py

```diff
@@ -140,9 +140,11 @@ def hull_area(par, llhs, above_min=1):
     float
     """
     min_llh = llhs.min()
-    Hull = ConvexHull(np.stack([par, llhs]).T[llhs < min_llh+above_min])
-    
-    return Hull.volume
+    try:
+        Hull = ConvexHull(np.stack([par, llhs]).T[llhs < min_llh+above_min])
+        return Hull.volume
+    except:
```

Comment:
Handle exceptions explicitly! Bare `except` are not allowed, you must specify the expected exception, e.g. `except AttributeError`